src/components/container/App/App.js

```diff
@@ -1,8 +1,10 @@
+import CandidateSection from '../../Components/CandidateSection/CandidateSection';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
+      <CandidateSection/>
```

Comment:
I guess, we need to keep the App.js in its initial state (empty) so far

Reply:
yes, please